source/material_model/visco_plastic.cc

```diff
@@ -216,6 +216,17 @@ namespace aspect
               out.thermal_conductivities[i] = MaterialUtilities::average_value (volume_fractions, thermal_conductivities, MaterialUtilities::arithmetic);
             }
 
+          /* In the following, the thermal conductivity is computed, if this option was selected.
```

Comment:
```suggestion
          /* Adapt the thermal conductivity to approximate hydrothermal cooling, if this option was selected.
```
Also, for one-line comments, you can use `//`

Reply:
I think here another comment is in order that state that the depth that the geometry model returns is not the actual depth below the subsurface, but the depth below the initial surface of the model domain.  Therefore, it will not be correct when mesh deformation is used (e.g. free surface).